src/SprykerEco/Zed/Payone/Business/Payment/MethodSender/PayoneCaptureMethodSender.php

```diff
@@ -0,0 +1,165 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Payone\Business\Payment\MethodSender;
+
+use Generated\Shared\Transfer\CaptureResponseTransfer;
+use Generated\Shared\Transfer\PayoneCaptureTransfer;
+use Generated\Shared\Transfer\PayoneStandardParameterTransfer;
+use Orm\Zed\Payone\Persistence\SpyPaymentPayoneApiLog;
+use SprykerEco\Zed\Payone\Business\Api\Adapter\AdapterInterface;
+use SprykerEco\Zed\Payone\Business\Api\Request\Container\Capture\BusinessContainer;
+use SprykerEco\Zed\Payone\Business\Api\Response\Container\CaptureResponseContainer;
+use SprykerEco\Zed\Payone\Business\Api\Response\Mapper\CaptureResponseMapperInterface;
+use SprykerEco\Zed\Payone\Business\Distributor\OrderPriceDistributorInterface;
+use SprykerEco\Zed\Payone\Business\Payment\DataMapper\ExpenseMapperInterface;
+use SprykerEco\Zed\Payone\Business\Payment\DataMapper\PayoneRequestProductDataMapperInterface;
+use SprykerEco\Zed\Payone\Business\Payment\DataMapper\StandartParameterMapperInterface;
+use SprykerEco\Zed\Payone\Business\Payment\PaymentMapperReaderInterface;
+use SprykerEco\Zed\Payone\Persistence\PayoneQueryContainerInterface;
+
+class PayoneCaptureMethodSender extends AbstractPayoneRequestSender implements PayoneCaptureMethodSenderInterface
+{
+    /**
+     * @see \Spryker\Shared\Shipment\ShipmentConstants::SHIPMENT_EXPENSE_TYPE
+     * @see \Spryker\Shared\Shipment\ShipmentConfig::SHIPMENT_EXPENSE_TYPE
+     *
+     * @deprecated Necessary in order to save compatibility with spryker/shipping version less than "^8.0.0".
```

Comment:
Fix the method document block.
* deprecation should go first.

Reply:
++